src/HookCoveredCallImplV1.sol

```diff
@@ -64,7 +67,9 @@ contract HookCoveredCallImplV1 is
   HookInstrumentERC721,
```

Comment:
are there functions within here, like on the OZ contracts, that pay attention to msg.Sender... do we need to update them?

Reply:
It seems likely that we want to allow only a fixed set of `external` functions call with msg.sender, but perhaps there are some like safeTransferFrom where that is not the case?